### DISCUSSION TITLE
Defining a default named transition for fragment morphs

### DISCUSSION THREAD

**Discussion by johndwells** [unpoly/unpoly#267] [Q&A] [answered]
I want to set `cross-fade` as the default transition when morphing between fragments. I was able to do this at one point but I discarded the code and now I can't find how in the docs. Something like `up.morph.config.transition = 'cross-fade';`... can anyone please advise?

**Comment by adam12** [comment, marked as answer]
```javascript
up.fragment.config.navigateOptions.transition = 'cross-fade'
```

From this slide in the Unpoly 2 presentation: http://triskweline.de/unpoly2-slides/#19

**Reply by johndwells** [reply]
I love the shade being thrown on that slide. :)
